improver_tests/utilities/test_cube_extraction.py

```diff
@@ -483,5 +488,191 @@ def test_single_threshold_use_original_units(self):
         )
 
 
+class Test_subset_data(IrisTest):
+    """Tests for the subset_data function"""
```

Comment:
And for the "private" method `_create_cutout` upon which it depends. If this is to remain some kind of private method, it probably needs to recorded in this doc-string that the tests here cover its behaviour as the two are not bound together within a plugin.

Reply:
I am now tempted to define `_create_cutout` inside the function's scope.  That might be cleaner.